root/src/main/java/com/example/servingwebcontent/MovieListController.java

```diff
@@ -55,6 +56,32 @@ public String test() throws java.io.FileNotFoundException {
 
 		connection.closeConnection();
 
+		return "test";
+	}**/
+
+	@GetMapping(value = "/test/{movieId}")
+	public String test (@PathVariable String movieId) throws java.io.FileNotFoundException {
+		BufferedReader reader;
+		String user = "", pass = "";
+	
+		try {
+			reader = new BufferedReader (new FileReader("./userInfo.txt"));
+			String line = reader.readLine();
+			user = line;
+			line = reader.readLine();
+			pass = line;
+			reader.close();
+		} catch (IOException e) { throw new RuntimeException("user info file not found" ,e); }
+
+		ConnectDatabase connection = new ConnectDatabase("movielist", user, pass);
+
+		try {
+			ResultSet rs = connection.queryDB("SELECT * from movies WHERE movieid = '" + movieId + "'");
```

Comment:
I realized that you didn't use prepared statements for the queryDB method so it's vulnerable to SQL injection, please change it to use prepared statements.